.grit/patterns/re_match_walrus.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Walrus operator for match in if
+---
+
+Use the walrus operator for snippets with the pattern:
+
+```python
+# Convert this:
+math = re.match(regex):
+if match:
+   ...
+
+# To this:
+if math := re.match(regex):
+   ...
+```
```

Comment:
We should not have these sorts of snippets in descriptions. This is actually the exact point of the samples *embedded* in the markdown.

You can move this down below as the first sample for this pattern, and that will make it automatically tested.